### PR TITLE
Update glueviz version to 0.10.0

### PR DESCRIPTION
**Pull request by bsipocz**

### CODE REVIEW

File: requirements.yml [hunk 1]
```diff
@@ -190,9 +190,9 @@
   setup_options: '--offline'
   include_extras: false
 
-#- name: glueviz
-#  version: '0.9.1'
-#  python: '<3.6*'
+- name: glueviz
+  version: '0.10.0'
+  python: '<3.6*'
```
Comment (astrofrog) [inline requirements.yml]: Are you getting the build from conda-forge or building your own here? 0.10.0 should work with Python 3.6

Comment (bsipocz) [reply requirements.yml]: You're right. Though I think that version limitation in ``requirements.yml`` is ignored at the moment (thus the reproject failure).

**Comment by bsipocz** [comment]
@mwcraig - something is broken again, glue should have been picked up from conda-forge, yet it fails somewhere in the middle.

**Comment by mwcraig** [comment]
@bsipocz -- I think the fail is on python 3.4. Can you update the python build restriction in `.travis.yml` and `appveyor.yml` to put the smallest python version at 3.5?

**Pull request merged**